### ISSUE TITLE
Cannot reproduce paper results on Real dataset (epoch_210 checkpoint + retraining), using DINOv2 depth_recovery

### ISSUE THREAD

**Issue by YangyiWAN** [CNJianLiu/MonoDiff9D#8]
Hi authors, thanks a lot for releasing MonoDiff9D and the pretrained checkpoints.

I’m trying to reproduce the reported performance on the Real dataset, but I’m unable to reach the paper’s numbers using both the released epoch_210 checkpoint and my own training. I’d appreciate any guidance on what might be missing.

What I did
1）Depth recovery
I generated depth for Real test images using the provided DINOv2 depth pipeline (NYU + DPT head, ViT-B/14).

Backbone: dinov2_vitb14_pretrain.pth
Head: dinov2_vitb14_nyu_dpt_head.pth
Config: dinov2_vitb14_nyu_dpt_config.py
Inference: model.whole_inference(batch, rescale=True)

1.Saved results as:
/mnt/e/Diff9d/data/Real/test/<scene>/dino_depth/<frame>.npy
(using np.save(depth_path, result.cpu()))

2）.Dataset loading changes
In provider/dataset.py, I modified depth loading from:
np.load(image_path + '_dinov2_large_depth.npy')
to:
np.load(os.path.join(os.path.dirname(image_path), 'dino_depth', os.path.basename(image_path) + '.npy'))
and also changed mrcnn_path to use self.data_dir (absolute path).

3.Evaluation results on Real (test)
Released checkpoint (epoch_210):3D IoU @25: 58.2、3D IoU @50: 26.3、3D IoU @75: 4.1、5° 2cm: 0.5、5° 5cm: 4.1、10° 2cm: 0.8、10° 5cm: 7.6
、10° 10cm: 23.1、10°: 57.6、10cm: 33.5
My retrained model:3D IoU @25: 56.0、3D IoU @50: 25.1、3D IoU @75: 4.3、5° 2cm: 0.6、5° 5cm: 3.7、10° 2cm: 1.1、10° 5cm: 7.2、10° 10cm: 22.2、10°: 59.1、10cm: 32.2
Both are significantly lower than the paper’s reported Real results.

**Questions**
1.Is the expected depth format/naming exactly _dinov2_large_depth.npy (and do you apply any scaling/normalization), rather than directly saving whole_inference output as .npy?
2.Are there any additional Real preprocessing steps (depth post-processing, intrinsics, fill_missing settings, etc.) required to reproduce the paper numbers?
3.Does the released epoch_210.pth correspond to a specific segmentation setting (e.g., trainedwithMask vs mrcnn), and what is the exact evaluation setup used in the paper?

Thanks again for your help!

Best regards,

[test_epoch210_logger.log](https://github.com/user-attachments/files/24352870/test_epoch210_logger.log)
[training_logger.log](https://github.com/user-attachments/files/24352869/training_logger.log)
[自己训练的_test_epoch210_logger.log](https://github.com/user-attachments/files/24352871/_test_epoch210_logger.log)
